docs/template_plugin/tests/functional/op_reference/region_yolo.cpp

```diff
@@ -0,0 +1,194 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+
+#include "openvino/op/region_yolo.hpp"
+#include "base_reference_test.hpp"
+
+using namespace reference_tests;
+using namespace ov;
+
+namespace {
+struct RegionYoloParams {
+    template <class IT>
+    RegionYoloParams(const size_t num,
+                    const size_t coords,
+                    const size_t classes,
+                    const bool do_softmax,
+                    const int axis,
+                    const int end_axis,
+                    const size_t batch,
+                    const size_t channels,
+                    const size_t width,
+                    const size_t height,
+                    const std::vector<int64_t>& mask,
+                    const ov::element::Type& iType,
+                    const std::vector<IT>& iValues,
+                    const std::vector<IT>& oValues,
+                    const std::string& testcaseName = "")
+        : num(num),
+          coords(coords),
+          classes(classes),
+          do_softmax(do_softmax),
+          axis(axis),
+          end_axis(end_axis),
+          batch(batch),
+          channels(channels),
+          width(width),
+          height(height),
+          mask(mask),
+          inType(iType),
+          outType(iType),
+          inputData(CreateTensor(iType, iValues)),
+          refData(CreateTensor(iType, oValues)),
+          testcaseName(testcaseName) {
+              inputShape = Shape{batch, channels, height, width};
+          }
+
+    size_t num;
+    size_t coords;
+    size_t classes;
+    bool do_softmax;
+    int axis;
+    int end_axis;
+    size_t batch;
+    size_t channels;
+    size_t width;
+    size_t height;
+    std::vector<int64_t> mask;
+    ov::PartialShape inputShape;
+    ov::element::Type inType;
+    ov::element::Type outType;
+    ov::runtime::Tensor inputData;
+    ov::runtime::Tensor refData;
+    std::string testcaseName;
+
+public:
+    template<typename T>
+    inline static std::vector<T> read_binary_file(const std::string& filename) {
+        std::string path = TEST_FILES + filename;
```

Comment:
Will be better to use https://github.com/openvinotoolkit/openvino/blob/master/src/common/util/include/openvino/util/file_util.hpp#L174

Reply:
Thank you for acknowledgement! I will use it.

Reply:
I used it. Thanks!